server-spi-private/src/main/java/org/keycloak/models/locking/LockAcquiringTimeoutException.java

```diff
@@ -22,9 +22,7 @@
 /**
  * This exception is thrown when acquiring a lock times out.
  */
-public final class LockAcquiringTimeoutException extends Exception {
-
-    private LockAcquiringTimeoutException() {}
+public final class LockAcquiringTimeoutException extends RuntimeException {
```

Comment:
Should we return the non-parametric constructor? This is to make sure the values `String lockName, String keycloakInstanceIdentifier, Instant timeWhenAcquired` are always set

Reply:
You are right. Sorry, it turned out I forgot how the no-argument constructor behaves when there is some parameteric added :man_facepalming: